### DISCUSSION TITLE
Audiobook Info is broken?

### DISCUSSION THREAD

**Discussion by TheRealMathemAddicts** [openaudible/openaudible#1248] [Q&A]
One of the key features of this app is completely missing for me. The "Audiobook Info" pane is completely unfilled for all of my audiobooks despite having been filled in previous versions. I just updated to 3.8.1.

I have gone through a couple of audiobooks, right-clicked, clicked Get Info, then Save (to save anything), but nothing gets populated in the Audiobook Info pane. Is anyone else having this issue?

**Comment by openaudible** [comment]
Make sure you download the books using openaudible and do a full rescan. That should get all the proper meta data. 

There is an "Auto-fill" feature that lets you add the ASIN and click autofill to get all the public info about the book.

**Reply by letternews** [reply]
when I use autofill the title and short title Get delete. I have to do the procedure again to add them back

**Comment by Cfomodz** [comment]
### Method 1: Full Rescan (Recommended)
1. Download books using OpenAudible (not manually from Audible website)
2. Go to **Library > Full Rescan**
3. Wait for the scan to complete
4. Verify Audiobook Info pane now shows metadata

### Method 2: Auto-fill Missing Books
1. Right-click the audiobook
2. Select **Get Info**
3. Add the ASIN (found in Audible URL: `https://www.audible.com/pd/Title/B00ABC1234`)
4. Click **Auto-fill**
5. Click **Save**

**Note:** If Auto-fill deletes the title, re-enter it manually after auto-filling.
